proxies/widdershins/options.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
it might be better if all of this (including the spec etc.) went in a `docs/` folder at the root of the repo. It's a bit confusing to have a bunch of folders in here named after different SDKs then just have `widdershins` in there as well